src/main/java/com/turbo/controller/OrdersController.java

```diff
@@ -0,0 +1,57 @@
+package com.turbo.controller;
+
+import com.turbo.model.Order;
+import com.turbo.service.OrderService;
+import com.turbo.service.PartService;
+import com.turbo.service.ReasonService;
+import com.turbo.service.TurbochargerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.validation.Valid;
+
+@Controller
+public class OrdersController {
+
+    private TurbochargerService turbochargerService;
+    private PartService partService;
+    private ReasonService reasonService;
+    private OrderService orderService;
+
+    @Autowired
+    public OrdersController(TurbochargerService turbochargerService, PartService partService, ReasonService reasonService, OrderService orderService) {
+        this.turbochargerService = turbochargerService;
+        this.partService = partService;
+        this.reasonService = reasonService;
+        this.orderService = orderService;
+    }
+
+
+    @GetMapping("/create-order")
+    public String createOrder(Model model) {
+        model.addAttribute("orderTurbos", turbochargerService.getAllTurbos());
+        model.addAttribute("partTurbos", partService.getAllParts());
+        model.addAttribute("reasonsTurbos", reasonService.getAllReasons());
+        model.addAttribute("order", new Order());
+        return "newOrder";
+    }
+
+    @PostMapping("/createNewOrder")
```

Comment:
Czy dodanie jako postfix do słowa Create słowa NewOrder coś zmieni? URI będzie http://domena.com/orders/createNewOrder. Jeśli zrezygnujesz ze słów NewOrder to zostanie http://domena.com/orders/create - jedno i drugie jest zrozumiałe, ale drugie jest krótsze. Decyzję pozostawiam Tobie